ga4gh/datamodel/variants.py

```diff
@@ -269,6 +269,7 @@ class HtslibVariantSet(datamodel.PysamDatamodelMixin, AbstractVariantSet):
     """
     def __init__(self, id_, dataDir):
         super(HtslibVariantSet, self).__init__(id_)
+        self._dataDir = dataDir
```

Comment:
Perhaps we should just move this up the inheritance chain, and add some getters?